pkg/sarif_test.go

```diff
@@ -852,3 +853,79 @@ func TestSARIFOutput(t *testing.T) {
 		})
 	}
 }
+
+func Test_createSARIFRuns(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		runs map[string]*run
+	}
+	tests := []struct {
+		name string
+		args args
+		want []run
+	}{
+		{
+			name: "empty runs",
+			args: args{
+				runs: map[string]*run{},
+			},
+			want: []run{},
+		},
+		{
+			name: "test with nil run also",
```

Comment:
optional/nit: If there's a purpose for a test, I prefer to see the reason instead of a description of the test which can be seen from the args.

```go
name: "nil runs are skipped"
```

Reply:
Fixed it. 👍